js/ui/lookup.js

```diff
@@ -518,10 +553,8 @@ const Lookup = DropDownList.inherit({
     },
 
     _renderPopup: function() {
-        if(this.option('usePopover')) {
-            if(this.option('_scrollToSelectedItemEnabled') && !this.option('itemCenteringEnabled') || !this.option('fullScreen')) {
```

Comment:
Why we remove check for the `_scrollToSelectedItemEnabled` option?

Reply:
fixed